benchmark/benchmark.py

```diff
@@ -30,6 +31,16 @@ def get_args() -> argparse.Namespace:
     parser_clean.add_argument('tag', type=str, help='tag for the machines', default='b2')
     parser_clean.set_defaults(func=clean)
 
+    parser_flooder = subparsers.add_parser('flooder')
+    parser_flooder.add_argument('--flooder-binary', type=Path, help='flooder executable')
+    group = parser_flooder.add_mutually_exclusive_group()
+    group.add_argument('--phrase', type=str, help='secret phrase : a path to a file or passed on stdin')
+    group.add_argument('--seed', type=str, help='secret seed of the account keypair passed on stdin')
+    parser_flooder.add_argument('--nodes', type=str, nargs='+', help='URL address(es) of the nodes to send transactions to')
+    parser_flooder.add_argument('--transactions', type=int, help='how many transactions to send')
```

Comment:
for ease, I would add some short alias, like `--tx`

Reply:
done